### PR TITLE
Break the process if the lock has been released outside of the process

### PR DESCRIPTION
**Pull request by clement-michelet**

### CODE REVIEW

File: Command/ProcessEventCommand.php [hunk 1]
```diff
@@ -123,28 +117,51 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+
             try {
-                while ($this->channels[$channel]['events_limit_per_run'] === null
-                    || $processedEventsCount < $this->channels[$channel]['events_limit_per_run']
+                while (
+                    $maxProcessedQueueSize === null
+                    || $processedEventsCount < $maxProcessedQueueSize
                 ) {
-                    if (!$event = $this->eventRepository->findFirstTodoEvent(false, $this->channels[$channel]['include'], $this->channels[$channel]['exclude'])) {
+                    if ($this->lockHandler->isLocked($channel)) {
```
Comment (juchi) [inline Command/ProcessEventCommand.php]: i think the condition needs to be reversed

Comment (clement-michelet) [inline Command/ProcessEventCommand.php]: Fixed

**Comment by juchi** [comment]
:+1:

**Comment by glenouvel** [comment]
:+1:

**Pull request merged**